cmd/acra-connector/acra-connector.go

```diff
@@ -397,7 +397,18 @@ func main() {
 	if connectorMode == connector_mode.AcraServerMode {
 		if *useTLS {
 			log.Infof("Selecting transport: use TLS transport wrapper")
-			tlsConfig, err := network.NewTLSConfig(network.SNIOrHostname(*tlsAcraserverSNI, *acraServerHost), *tlsCA, *tlsKey, *tlsCert, tls.ClientAuthType(*tlsAuthType))
+
+			ocspConfig, err := network.NewOCSPConfig("", "yes", "prefer")
```

Comment:
Currently, there is a stub in connector to make TLS config creation pass, but w/o CLI args parsing as in acra-server.

That's because I've noticed that even with TLS enabled in tests, `get_connector_tls_params()` is not executed.
See test class `SSLPostgresqlMixin`, it force defines `with_tls()` to return `False`. I've only tested on job `postgresql-ssl` and it seems to be the only one that redefines `with_tls()`